lib/packer/client.rb

```diff
@@ -40,14 +40,18 @@ def build(template, options = {})
 
       args << template
 
-      Packer::Output::Build.new(command(args))
+      Packer::Output::Build.new(
+        command(args, options.fetch(:stream_output, false)))
     end
 
     # @api private
     # @param [Array<String>] args to pass to Packer
-    def command(args)
+    # @param Boolean set to true to stream output to $stdout
+    def command(args, stream = false)
```

Comment:
Rather than a boolean to toggle streaming to stdout, how about a keyword argument that allows any IO object to be passed in, including `$stdout`, and defaults to `nil` to disable live streaming.

Reply:
Actually, that makes a lot more sense.  I'll get it updated today.